utils/src/time.rs

```diff
@@ -224,4 +224,7 @@ mod tests {
 
         assert!(seconds_to_nanoseconds(9_223_372_037).is_none());
     }
+
+    #[test]
+    fn test_stdin() {}
```

Comment:
What's up with this one?

Reply:
Upsy, just a silly leftover. Removed it. Take a look here:  [
1568750](https://github.com/firecracker-microvm/firecracker/pull/1413/commits/1568750cd716861296faee66063ec64f9cb722a3).